W3ChampionsStatisticService/Matches/MatchRepository.cs

```diff
@@ -241,4 +242,9 @@ public Task DeleteOnGoingMatch(string matchId)
         return _cache.CountOnGoingMatches(gameMode, gateWay, map, minMmr, maxMmr);
     }
 
+    public Task<Season> LoadLastSeason()
```

Comment:
Can you query the Season collection for the current season? Here's what it contains.
```
W3Champions-Statistic-Service> db.Season.find()
[
  { _id: 0 },  { _id: 1 },
  { _id: 2 },  { _id: 3 },
  { _id: 4 },  { _id: 5 },
  { _id: 6 },  { _id: 7 },
  { _id: 8 },  { _id: 9 },
  { _id: 10 }, { _id: 11 },
  { _id: 12 }, { _id: 13 },
  { _id: 14 }, { _id: 15 },
  { _id: 16 }, { _id: 17 },
  { _id: 18 }
]
```

Reply:
sorry, didn't understand, could you please add some details?
```c#
mongoCollection.AsQueryable().OrderByDescending(c => c.Id).FirstOrDefaultAsync();
```
returns only one object, as far I understand

Reply:
I think I'm just C# illiterate! This already queries the Season collection, right?

Reply:
yeah